lib/decompress.ps1

```diff
@@ -286,34 +231,7 @@ function Expand-ZipArchive {
         $OriDestinationPath = $DestinationPath
         $DestinationPath = "$DestinationPath\_tmp"
     }
-    # All methods to unzip the file require .NET4.5+
-    if ($PSVersionTable.PSVersion.Major -lt 5) {
-        Add-Type -AssemblyName System.IO.Compression.FileSystem
-        try {
-            [System.IO.Compression.ZipFile]::ExtractToDirectory($Path, $DestinationPath)
-        } catch [System.IO.PathTooLongException] {
-            # try to fall back to 7zip if path is too long
-            if (Test-HelperInstalled -Helper 7zip) {
-                Expand-7zipArchive $Path $DestinationPath -Removal
-                return
-            } else {
-                abort "Unzip failed: Windows can't handle the long paths in this zip file.`nRun 'scoop install 7zip' and try again."
-            }
-        } catch [System.IO.IOException] {
-            if (Test-HelperInstalled -Helper 7zip) {
-                Expand-7zipArchive $Path $DestinationPath -Removal
-                return
-            } else {
-                abort "Unzip failed: Windows can't handle the file names in this zip file.`nRun 'scoop install 7zip' and try again."
-            }
-        } catch {
-            abort "Unzip failed: $_"
-        }
-    } else {
-        # Use Expand-Archive to unzip in PowerShell 5+
-        # Compatible with Pscx (https://github.com/Pscx/Pscx)
-        Microsoft.PowerShell.Archive\Expand-Archive -Path $Path -DestinationPath $DestinationPath -Force
-    }
+    Expand-Archive -Path $Path -DestinationPath $DestinationPath -Force
```

Comment:
You removed the `Microsoft.PowerShell.Archive\` prefix, which was added to fix the confliction with Pscx ...

Reply:
Pscx change their function name to `Expand-PscxArchive` ^_^

Reply:
Keep the "Microsoft.PowerShell.Archive" prefix does not have any cons isn't it?

Reply:
Under this release [4.0.0-beta1 - October 17, 2020](https://github.com/Pscx/Pscx/blob/master/ReleaseNotes.txt)

Reply:
I'm afraid this may break something in *nix environment since the `\`